addons/account_asset/__openerp__.py

```diff
@@ -15,7 +15,7 @@
     'category': 'Accounting',
     'sequence': 32,
     'demo': [
-        'account_asset_demo.yml',
+        'test/account_asset_demo.yml',
```

Comment:
If it is a demo file, it should be in the data directory. Even if it is used in tests.